docs/sources/panels-visualizations/configure-standard-options/index.md

```diff
@@ -25,84 +25,94 @@ weight: 60
 
-You can also paste a native emoji in the unit picker and pick it as a custom unit:
+| Custom unit                        | Description                                                                                                                                                                                                    |
+| ---------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `suffix:<suffix>`                  | Custom unit that should go after value.                                                                                                                                                                        |
+| `prefix:<prefix>`                  | Custom unit that should go before value.                                                                                                                                                                       |
+| `time:<format>`                    | Custom date time formats type, such as `time:YYYY-MM-DD`. Refer to [formats](https://momentjs.com/docs/#/displaying/) for the format syntax and options.                                                       |
+| `si:<base scale><unit characters>` | Custom SI units, such as `si: mF`. You can specify both a unit and the source data scale. For example, if your source data is represented as milli-something, prefix the unit with the `m` SI scale character. |
+| `count:<unit>`                     | Custom count unit.                                                                                                                                                                                             |
+| `currency:<unit>`                  | Custom currency unit.                                                                                                                                                                                          |
 
-{{< figure src="/static/img/docs/v66/custom_unit_burger2.png" max-width="600px" caption="Custom unit emoji" >}}
+You can also paste a native emoji in the **Unit** drop-down and select it as a custom unit:
+
+![A thumbs up emoji as a custom unit](/media/docs/grafana/panels-visualizations/custom_unit_thumbsup_v11.0.png)
+
+![A time series visualization using custom thumbs up emoji units](/media/docs/grafana/panels-visualizations/thumbsup_panel_v11.0.png)
 
 #### String units
 
-Grafana can sometimes be too aggressive in parsing strings and displaying them as numbers. To configure Grafana to show the original string value, create a field override and add a unit property with the `String` unit.
+Sometimes Grafana is too aggressive in parsing strings and displaying them as numbers. To configure Grafana to show the original string value, select **Misc > String** in the **Unit** drop-down.
```

Comment:
Maybe _interpreting_ would make more sense than _parsing_? Seems a bit less technical, but certainly not my area of expertise here 😄

Reply:
I like it. Will do!